install/kubernetes/helm/istio/charts/mixer/templates/deployment.yaml

```diff
@@ -15,13 +15,11 @@ spec:
   template:
     metadata:
       labels:
-        app: {{ template "mixer.name" . }}
-        release: {{ .Release.Name }}
         istio: mixer
```

Comment:
I think these labels are helm convention you may want to keep.

Reply:
I tried. We can add them later ( but will break upgrade ). 

k8s treats the label set as immutable for some strange reasons.